### DISCUSSION TITLE
Tethered Wing data extraction

### DISCUSSION THREAD

**Discussion by aweproject2023** [byuflowlab/FLOWUnsteady#67] [Q&A] [answered]
i am wondering how I can extract raw data (text or cvs) of these plots generated from tethered wing example? 

![image](https://github.com/byuflowlab/FLOWUnsteady/assets/131621764/2fa4d05c-3f7a-4387-92f3-c814426037d7)

**Comment by aweproject2023** [comment]
The following script works for me for wing-example case raw data extraction. But for tethered wing case in circular path, I am wondering is it possible to extract wing loading for every half circle? Thx!

#push!(CLs, CL)
#push!(CDs, CD)

#push!(rolls, roll)
#push!(pitchs, pitch)
#push!(yaws, yaw)

push!(spanposs, spanpos)
push!(ls, l)
push!(ds, d)

f = open("rawdata_lift.txt", "w")
println(f, ls)
close(f)

f = open("rawdata_span.txt", "w")
println(f, spanposs)
close(f)

**Comment by EdoAlvarezR** [comment, marked as answer]
Similar to this thread: [LINK](https://github.com/byuflowlab/FLOWUnsteady/discussions/61)

In order to determine the half circle I'd suggest applying some logic based on the current step number.
